### ISSUE TITLE
add the option to import the seed

### ISSUE THREAD

**Issue by joticajulian** [joticajulian/kondor#16]
BIP32 derivation paths would look like this `m/44'/659'/0'/0`
The number 659 is the proposed coin type for koinos. It is the word "koinos" converted to ascii [107 111 105 110 111 115], and then added. The result is 659 xD.

**Comment by Yonatan19981** [comment]
Can we use this code and ethers to combine the imported seed with the existing code for private key?: 

const ethers = require('ethers');
let mnemonic = "YOUR MNEMONIC";
let mnemonicWallet = ethers.Wallet.fromMnemonic(mnemonic);
console.log(mnemonicWallet.privateKey);

**Comment by joticajulian** [comment]
the function `ethers.Wallet.fromMnemonic` uses the path for ethereum (`m/44'/60'/0'/0`). And we should a different one.

The code to specify our custom path is something like this:
```
const ethers = require("ethers");
const hdNode = ethers.utils.HDNode.fromMnemonic("heart spot flock era duck indoor myself summer expand extend habit hint");
const keyNumber0 = hdNode.derivePath("m/44'/659'/0'/0/0");
const keyNumber11 = hdNode.derivePath("m/44'/659'/0'/0/11");
console.log(keyNumber11);

// HDNode {
//     privateKey: '0xaa76e7f83c6a6714305773b8ee10e34a4ff93c0d8f7ba8e3fc63ebb8a7a09695',
//     publicKey: '0x02e27d11a924d7508122f6e07d0256fd2a549cc44303af23ce067a203fa72024ee',
//     parentFingerprint: '0xce1d259a',
//     fingerprint: '0xed6af953',
//     address: '0x375e0e15c2786dEe73fbFf8854e18C4975e10Eac',
//     chainCode: '0x64a9c7992261beb244bd73d99771b47af545eaa77e75f1d3a748c7af94731b3d',
//     index: 11,
//     depth: 5,
//     mnemonic: {
//       phrase: 'heart spot flock era duck indoor myself summer expand extend habit hint',
//       path: "m/44'/659'/0'/0/11",
//       locale: 'en'
//     },
//     path: "m/44'/659'/0'/0/11"
//   }
```

**Comment by joticajulian** [comment]
we can import ethers for now. And later we can change to a specific library for this feature (to not import all ethers lib since it has several dependencies that we will not use)

**Comment by Yonatan19981** [comment]
Pushed update with relevant UI and code

**Comment by joticajulian** [comment]
thanks! could you create a pull request?

**Comment by joticajulian** [comment]
solved in #27
